src_assets/common/assets/web/configs/tabs/AudioVideo.vue

```diff
@@ -69,6 +69,18 @@ const config = ref(props.config)
         :config="config"
     />
 
+    <PlatformLayout :platform="platform">
+      <template #windows>
+        <!-- GPU Preference -->
+        <div class="mb-3">
+          <label for="gpu_preference" class="form-label">{{ $t('config.gpu_preference') }}</label>
+          <input type="text" class="form-control" id="gpu_preference" placeholder="-1"
```

Comment:
Can use input type number. Maybe do -1 to 32 or 64? (or we don't have to set a max)

https://github.com/LizardByte/Sunshine/blob/9b9767be0c7f8f169029c3fa1f5ce92dd5ea416c/src_assets/common/assets/web/configs/tabs/encoders/NvidiaNvencEncoder.vue#L54